README.md

```diff
@@ -16,8 +16,14 @@ Please refer to documentation of each individual component for download and setu
 
 ### RabbitMQ connection string
 
-RabbitMQ connection string can be set using `Backend.RabbitMqConnectionString.Value` property in the `Shared` project.
+RabbitMQ connection string can be configured by setting the `Value` property of the `Backend.RabbitMqConnectionString` class in the `Shared` project.
```

Comment:
@SeanFeldman I fixed it on all branches. Can you check it compiles and still works?